.gitignore

```diff
@@ -35,44 +34,44 @@ pip-log.txt
 *.lprof
 nosetests.xml
 /src/odemis/acq/align/test/hfw_shift*.tiff
+/src/odemis/acq/align/test/res_shift.tiff
 /src/odemis/acq/align/test/twin_stage*.tiff
 /src/odemis/acq/test/test*.tiff
+/src/odemis/cli/test/test.tiff
+/src/odemis/gui/test/test.ome.tiff
+/src/odemis/gui/util/test/test.ome.tiff
+/src/odemis/util/test/test.ome.tiff
+/src/odemis/gui/test/test.png
 
 # Dev environment
 odemis.sublime-project
 odemis.sublime-workspace
-/src/odemis/gui/test/test.png
+*.autosave
+odemis.profile
+magic.odm.yaml
+*.idea
 .project
 .pydevproject
 .settings
 
-#Translations
+# Translations
 *.mo
 
-#Rinze
-launch_daemon
-xrced
-swap.sh
-*.autosave
-odemis.profile
-magic.odm.yaml
-*.idea
-
 # documentation files
-doc/code/_build
-doc/code/_gen
-doc/develop/_build/
+/doc/code/_build
+/doc/code/_gen
+/doc/develop/_build/
+/doc/develop/SPARC*.pdf
+/doc/develop/*-roles.pdf
+/doc/develop/gui_cont.pdf
```

Comment:
I have gui_tree.pdf from when I built the docs at the beginning of January, did gui_tree.pdf get renamed to gui_cont.pdf?